backend/app/db_accessors/accessor.js

```diff
@@ -0,0 +1,44 @@
+import Connection from "../db/connection.js";
+import Blog from "../models/blog.js";
+
+export default class Accessor {
+    static async getAllBlogs() {
+        try {
+            await Connection.open("teaching_proj");
+            const messages = await Message.find({});
+            return messages;
+        } catch (e) {
+            console.log("Something bad will happen");
+        }
```

Comment:
I know that once again I probably wrote these console.logs originally so sorry about that but it would be good practice to have all the console.log messages be the same and have them all show what the error message is